packages/webcomponents/src/ui-components/styled-host/styled-host.tsx

```diff
@@ -2,13 +2,11 @@ import { FunctionalComponent, HostAttributes } from "@stencil/core/internal";
 import { Host, h } from '@stencil/core';
 import modifiedBootstrap from './modified-bootstrap.css';
 
-const StyledHost: FunctionalComponent<HostAttributes> = (props, children) => {
+export const StyledHost: FunctionalComponent<HostAttributes> = (props, children) => {
   return (
-    <Host {...props} class="body">
+    <Host {...props} class="body" style={{ display: 'block' }}>
```

Comment:
this file is doing so much heavy lifting for us now. Great job with these abstractions!